src/frontend/components/LiveVideoPanel/LiveVideoTabPanel/index.spec.tsx

```diff
@@ -80,10 +79,9 @@ describe('<LiveVideoTabPanel /> styles', () => {
     expect(button).toHaveStyle('flex: 1;');
     expect(button).toHaveStyle('box-shadow: inset 0 -2px #055fd2;');
 
-    const text = screen.getByRole('tab_title');
+    const text = screen.getByTestId('tab_title');
```

Comment:
```suggestion
    const text = screen.getByText('application');
```

You don't need this `test-id` at all and indeed the `getByRole` was not used as it should be.

Reply:
Also you should add a selector to your `getByRole` test.

```
const button = screen.getByRole('tab', {name: 'application');
```

Reply:
2ad2345